boards/st/stm32l562e_dk/doc/index.rst

```diff
@@ -273,6 +276,15 @@ Serial Port
 STM32L562E-DK Discovery board has 6 U(S)ARTs. The Zephyr console output is
 assigned to USART1. Default settings are 115200 8N1.
 
+TFT LCD screen and touch panel
+------------------------------
+
+The TFT LCD screen and touch panel are supported for the STM32L562E-DK Discovery board.
+They can be tested using :zephyr:code-sample:`lvgl` sample:
+
+.. code-block:: console
+
+   $ west build -b stm32l562e_dk samples/subsys/display/lvgl/
```

Comment:
this should actually be using `.. zephyr-app-commands::` rather than hard-coding the build command. This also makes it easier to catch stale build instructions should e.g the sample be moved or whatever

```suggestion
.. zephyr-app-commands::
   :zephyr-app: samples/subsys/display/lvgl
   :board: stm32l562e_dk
   :goals: build
```

I should have also reviewed the PR more thoroughly when asking you to rebase, sorry to be asking you for another update

Reply:
I made the changes.
The other commands in this docs are also hardcoded and should be refactored.